packages/cli/src/actions/modify.ts

```diff
@@ -3,6 +3,7 @@ import { createRequire } from "node:module";
 import { cp } from "node:fs/promises";
 import { fileURLToPath } from "node:url";
 
+import { Command } from "commander";
```

Comment:
Did you forget to add Commander to the package.json?

Reply:
Mb..